frontend/src/static/js/v_authorship.js

```diff
@@ -111,12 +118,12 @@ window.vAuthorship = {
           const out = {};
           out.path = file.path;
           out.lineCount = lineCnt;
-          this.addLineCountToFileType(file.path, lineCnt, filesInfoObj);
 
           const segmentInfo = this.splitSegments(file.lines);
           out.segments = segmentInfo.segments;
           totalBlankLineCount += segmentInfo.blankLineCount;
-          this.addLineCountToFileType(file.path, segmentInfo.blankLineCount, filesBlanksInfoObj);
+          this.addBlankLineCountToFileType(file.path, segmentInfo.blankLineCount,
+            filesBlanksInfoObj);
```

Comment:
double indentation
@ongspxm seems like this is not covered by eslint, can you open a PR to fix it?

Reply:
@eugenepeh CI is failing due to this double indentation which you asked me to add